include/robot_interfaces/sensors/pylon_driver.hpp

```diff
@@ -30,15 +30,28 @@ class PylonDriver : public SensorDriver<CameraObservation>
 public:
     Pylon::PylonAutoInitTerm auto_init_term_;
     Pylon::CInstantCamera camera_;
+    Pylon::CTlFactory& tl_factory_;
+    Pylon::DeviceInfoList_t device_list_;
+    Pylon::DeviceInfoList_t::const_iterator device_iterator_;
```

Comment:
It seems that these variables are only used in the constructor? If so, please move them there.